app/uk/gov/hmrc/nisp/controllers/CustomLanguageController.scala

```diff
@@ -36,4 +37,11 @@ class CustomLanguageController @Inject()(implicit override val messagesApi: Mess
   /** Returns a mapping between strings and the corresponding Lang object. **/
   override def languageMap: Map[String, Lang] = Map("english" -> englishLang,
     "cymraeg" -> Lang("cy"))
+
+  def switchToWelshLandingPage: Action[AnyContent] = Action { implicit request =>
+    val enabled =  application.configuration.getBoolean("microservice.services.features.welsh-translation").getOrElse(true)
+    val lang =  if (enabled) Lang("cy")
```

Comment:
Can you refactor this use the languageMap? Or perhaps refactor them both to use `englishLang` and `welshLang`